app/scripts/controllers/MainCtrl.js

```diff
@@ -1,9 +1,21 @@
 (function () {
-	function MainCtrl($scope, Room) {
+	function MainCtrl($scope, Room, Message) {
 		$scope.rooms = Room.all;
+
```

Comment:
I'd suggest adding `$scope.activeRoom = null;` to the function so the property is defined on page load.